src/leakybucket/__init__.py

```diff
@@ -0,0 +1,5 @@
+__version__ = "0.0.1"
+
+from leakybucket import leakybucket_bmi
+
+__all__ = ["leakybucket_bmi"]
```

Comment:
I would no re-export anything as you are not using it anywhere. Also this makes any importing submodules impossible to import without also importing leakybucket_bmi module.

```suggestion
```